### ISSUE TITLE
License

### ISSUE THREAD

**Issue by bleetube** [getAlby/ldk-node-go#27]
Hello, is it safe to assume this repo is using the Apache 2.0 license like getAlby/ldk-node and getAlby/hub?
Asking because I'm attempting to create a [nix derivation](https://github.com/NixOS/nixpkgs/pull/368929) which needs to have the license for the ldk-node-go package declared.

**Comment by rolznz** [comment]
Hi, I think this is fine - because both LDK and Alby Hub can be treated as Apache 2.0.

https://github.com/lightningdevkit/ldk-node/blob/main/LICENSE.md
https://github.com/getAlby/hub/blob/master/LICENSE

**Comment by bleetube** [comment]
Thanks! I'll wait to close this until after I merge the package, just in case they insist there needs to be a LICENSE file.  If you happen to include the license file before then, feel free to close this issue!

**Comment by rolznz** [comment]
Yeah, maybe it's better to be explicit, thanks. I added one. https://github.com/getAlby/ldk-node-go/blob/master/LICENSE
